website/docs/r/servicebus_subscription.html.markdown

```diff
@@ -73,6 +73,18 @@ The following arguments are supported:
 
 * `status` - (Optional) The status of the Subscription. Possible values are `Active`,`ReceiveDisabled`, or `Disabled`. Defaults to `Active`.
 
+* `client_scoped_subscription_enabled` - (Optional)  whether the subscription has an affinity to the client id. Defaults to `False`. 
+
+~> **NOTE:** Client Scoped Subscription can only be used for JMS subscription.
+
+* `client_id_for_client_scoped_subscription` - (Optional)  Specifies the Client ID of the application that created the client-scoped subscription.
```

Comment:
is this client "scoped" or client affinity?
```suggestion
* `client_id` - (Optional)  The client id this subscription should have an affinity with.
```

the mixing of terms here is quite confusing